src/main/kotlin/no/nav/familie/ks/sak/common/util/Utils.kt

```diff
@@ -0,0 +1,8 @@
+package no.nav.familie.ks.sak.common.util
+
+object Utils {
+
+    fun Any?.nullableTilString() = this?.toString() ?: ""
+
+    fun String.storForbokstav() = this.lowercase().replaceFirstChar { it.uppercase() }
+}
```

Comment:
Trenger vi et objekt for dette?
```suggestion
fun Any?.nullableTilString() = this?.toString() ?: ""

fun String.storForbokstav() = this.lowercase().replaceFirstChar { it.uppercase() }
```